docs/src/howto/install/charm.md

```diff
@@ -0,0 +1,87 @@
+# Install Canonical Kubernetes from a charm
+
+Canonical Kubernetes is packaged as a [charm], available from the 
+snap store for all supported platforms.
```

Comment:
you mean charm store? I would probably also like one line about Juju and appropriate links